index.html

```diff
@@ -4,9 +4,107 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Tribute Page Freecodecamp</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Hello world</h1>
-</body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <figure id="img-div">
+<center><img src="Images/tribute-page-main-image.jpg" id="image" style="display:block;max-width:100%"></center>
+            <figcaption id="img-caption">
+                Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+            </figcaption>
+        </figure>
+        <center> <div id="tribute-info"><h3>Here's a time line of Dr. Borlaug's life:</h3>
+           
+            <ul>
+                <li><strong>1914</strong> - Born in Cresco, Iowa</li>
+                <li>
```

Comment:
- [ ] it is better to apply the bold text in an external CSS file rather than repeating the `<strong></strong>` tag (eg li{`font-weight: bold`})

Reply:
Kindly phrase your comments in a polite and considerate manner, like a suggestion, instead of a command. This will make it easy for the student to accept and implement them.

Reply:
OK Thank you